### PR TITLE
update package.json with both html and text coverage reports

### PR DESCRIPTION
**Pull request by alopezsanchez**
It can be useful to local testing.

https://istanbul.js.org/docs/advanced/alternative-reporters/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
     "test": "npm run test-unit && npm run lint",
     "test-unit": "mocha --require intelli-espower-loader test/unit/*.test.js",
     "test-all": "mocha --require intelli-espower-loader test/**/*.test.js",
-    "coverage": "nyc npm test",
-    "coverage-all": "nyc npm run test-all",
+    "coverage": "nyc --reporter=lcov --reporter=text --reporter=text-summary npm test",
```
Comment (joyeecheung) [inline package.json]: Maybe add `--reporter=html`?

Comment (alopezsanchez) [reply package.json]: Lcov generates both `html` and `lcov`. I assume that `lcov` is only useful for you to `codecov`.

I change it, no problem.

**Comment by joyeecheung** [comment]
@alopezsanchez Looks like there is a conflict now, can you rebase? thanks!

**Comment by alopezsanchez** [comment]
Conflict resolved and change made.

**Comment by apapirovski** [comment]
Hey @alopezsanchez — unfortunately I don't think this can get merged with the merge commit in here. If you do `git reset --hard upstream/master` (or whatever your upstream reference is called) and then just copy & paste your changes, commit & `git push --force` them that should give us a cleaner commit tree to work with.

Or you can:

```
git reset --hard upstream/master
git cherry-pick 8aef2ed44b6302deb6b09bbb6c9b32e404371bc7
git cherry-pick e216ce2b7edd5cf42c64ad39169bbc3d0a148c93
```

but I'm not sure if there are any irresolvable conflicts there, hence the copy & paste and create new commit suggestion.

**Comment by alopezsanchez** [comment]
Oh yes, you are right. I resolved the conflict from Github's page. Fatal error...
Sorry for the inconvenience, I'll fix it.

**Pull request merged**